apps/namehashlabs.org/components/2 - molecules/financial-support-section.tsx

```diff
@@ -22,21 +23,19 @@ export const FinancialSupportSection = () => {
           <Balancer>Our work has received generous support from:</Balancer>
         </div>
         <div className="flex gap-3 items-center justify-center">
-          <a
-            target="_blank"
-            aria-label="ENS"
-            href="https://ensdao.org/"
-            className="py-[9px] px-[25px] bg-white cursor-pointer hover:bg-gray-50 transition-colors duration-200 rounded-[8px] border border-gray-200"
-          >
-            <EnsLogoColored className="w-full" />
+          <a target="_blank" aria-label="ENS" href="https://ensdao.org/">
+            <Button variant="secondary" size="large">
+              <EnsLogoColored className="w-full h-7" />
```

Comment:
We should rename this UI component. It's not the ENS logo. It's the ENS DAO logo.